rust/oak/src/lib.rs

```diff
@@ -132,3 +137,65 @@ pub extern "C" fn oak_handle_grpc_call() {
         }
     });
 }
+
+pub fn execute_storage_operation(
```

Comment:
I would suggest moving the storage-related stuff in its own module to start with (so it can be invoked as `oak::storage::execute_storage_operation` or similar, unless previously imported)

Reply:
Done.